### PR TITLE
Modified script jul 29

### PR DESCRIPTION
**Pull request by manjunath-grl**
Modified scripts:
Fixes issue: [331](https://github.com/CHIP-Specifications/chip-test-scripts/issues/331)
TC-BINFO-2.1
TC-ILL-2.1
TC-LVL-3.1
TC-CC-5.1
TC-PC-2.1
TC-SWTCH-2.1
PICS update
TC-CC-3.*
TC-CC-4.*
TC-CC-5.*
TC-CC-6.*
TC-CC-7.*
TC-CC-9.*
TC-WNCV-3.*
TC-WNCV-4.*
Added auto generated files

Tested
Attached execution logs:
[Execution_log.txt](https://github.com/project-chip/connectedhomeip/files/9221462/Execution_log.txt)

### CODE REVIEW

**Review by woody-apple — APPROVED** [review]
Fast tracking test updates

File: src/app/tests/suites/certification/Test_TC_WNCV_3_1.yaml [hunk 1]
```diff
@@ -51,12 +51,10 @@ tests:
...
     ###
     ### Mandatory Depends on the FeatureMap
-    - label:
-          "1c: If (PA & LF) TH reads CurrentPositionLiftPercent100ths attribute
-          from DUT"
+    - label: "1c: TH reads CurrentPositionLiftPercent100ths attribute from DUT"
       command: "readAttribute"
       attribute: "CurrentPositionLiftPercent100ths"
-      PICS: WNCV.S.F00 && WNCV.S.F02 && WNCV.S.A000e
+      PICS: WNCV.S.A000e && ( WNCV.S.F00 || WNCV.S.F01 )
```
Comment (jmeg-sfy) [inline src/app/tests/suites/certification/Test_TC_WNCV_3_1.yaml:57]: this is wrong !

File: src/app/tests/suites/certification/Test_TC_WNCV_3_1.yaml [hunk 2]
```diff
@@ -65,24 +63,22 @@ tests:
...
     ### Optional Depends on the FeatureMap
     - label:
-          "1d: If (PA & LF) TH reads CurrentPositionLiftPercentage optional
-          attribute from DUT"
+          "1d: TH reads CurrentPositionLiftPercentage optional attribute from
+          DUT"
       command: "readAttribute"
       attribute: "CurrentPositionLiftPercentage"
-      PICS: WNCV.S.F00 && WNCV.S.F02 && WNCV.S.A0008
+      PICS: WNCV.S.A0008 && ( WNCV.S.F00 || WNCV.S.F01 )
```
Comment (jmeg-sfy) [inline src/app/tests/suites/certification/Test_TC_WNCV_3_1.yaml:70]: wrong

File: src/app/tests/suites/certification/Test_TC_WNCV_3_1.yaml [hunk 3]
```diff
@@ -91,11 +87,11 @@ tests:
...
     ### Optional Depends on the FeatureMap
     - label:
-          "1f: If (PA & TL) TH reads CurrentPositionTiltPercentage optional
-          attribute from DUT"
+          "1f: TH reads CurrentPositionTiltPercentage optional attribute from
+          DUT"
       command: "readAttribute"
       attribute: "CurrentPositionTiltPercentage"
-      PICS: WNCV.S.F01 && WNCV.S.F04 && WNCV.S.A0009
+      PICS: WNCV.S.A0009 && ( WNCV.S.F00 || WNCV.S.F01 )
```
Comment (jmeg-sfy) [inline src/app/tests/suites/certification/Test_TC_WNCV_3_1.yaml:94]: wrong

File: src/app/tests/suites/certification/Test_TC_WNCV_3_1.yaml [hunk 4]
```diff
@@ -183,12 +175,10 @@ tests:
...
 
     ### Mandatory Depends on the FeatureMap
-    - label:
-          "3b: If (PA & LF) TH reads CurrentPositionLiftPercent100ths attribute
-          from DUT"
+    - label: "3b: TH reads CurrentPositionLiftPercent100ths attribute from DUT"
       command: "readAttribute"
       attribute: "CurrentPositionLiftPercent100ths"
-      PICS: WNCV.S.F00 && WNCV.S.F02 && WNCV.S.A000e
+      PICS: WNCV.S.A000e && ( WNCV.S.F00 || WNCV.S.F01 )
```
Comment (jmeg-sfy) [inline src/app/tests/suites/certification/Test_TC_WNCV_3_1.yaml:181]: wrong

**Review by jmeg-sfy — CHANGES_REQUESTED** [review]
Pics definition/condition are wrong in many WNCV Test Case
this need to be thoroughly recheck
either drop WNCV update from this PR and push them in a consecutive one or really fix those

File: src/app/tests/suites/certification/Test_TC_WNCV_3_5.yaml [hunk 5]
```diff
@@ -77,48 +77,44 @@ tests:
...
     ### Step 3x -> Verify the DUT has reached its fully-open limits
     ### Mandatory Depends on the FeatureMap
-    - label:
-          "3a: If (PA & LF) TH reads CurrentPositionLiftPercent100ths attribute
-          from DUT"
+    - label: "3a: TH reads CurrentPositionLiftPercent100ths attribute from DUT"
       command: "readAttribute"
       attribute: "CurrentPositionLiftPercent100ths"
-      PICS: WNCV.S.F00 && WNCV.S.F02 && WNCV.S.A000e
+      PICS: WNCV.S.A000e && ( WNCV.S.F00 || WNCV.S.F01 )
```
Comment (jmeg-sfy) [inline src/app/tests/suites/certification/Test_TC_WNCV_3_5.yaml:90]: attribute CurrentPositionLiftPercent100ths is mandatory if LF & PA_LF -> WNCV.S.F00 &&  WNCV.S.F02

File: src/app/tests/suites/certification/Test_TC_WNCV_3_1.yaml (hunk 1 above)
Comment (manjunath-grl) [reply src/app/tests/suites/certification/Test_TC_WNCV_3_1.yaml]: Changes removed please verify

File: src/app/tests/suites/certification/Test_TC_WNCV_3_1.yaml (hunk 2 above)
Comment (manjunath-grl) [reply src/app/tests/suites/certification/Test_TC_WNCV_3_1.yaml]: Removed changes from PR please verify it.

File: src/app/tests/suites/certification/Test_TC_WNCV_3_1.yaml (hunk 3 above)
Comment (manjunath-grl) [reply src/app/tests/suites/certification/Test_TC_WNCV_3_1.yaml]: Removed changes from PR please verify it.

File: src/app/tests/suites/certification/Test_TC_WNCV_3_1.yaml (hunk 4 above)
Comment (manjunath-grl) [reply src/app/tests/suites/certification/Test_TC_WNCV_3_1.yaml]: Removed changes from PR please verify it.

File: src/app/tests/suites/certification/Test_TC_WNCV_3_5.yaml (hunk 5 above)
Comment (manjunath-grl) [reply src/app/tests/suites/certification/Test_TC_WNCV_3_5.yaml]: Removed changes from PR please verify it.

**Pull request merged**